src/pydata_sphinx_theme/assets/styles/components/_navbar-links.scss

```diff
@@ -21,24 +21,22 @@
       display: flex;
       align-items: center;
       height: 100%;
-      color: var(--pst-color-text-muted);
       padding-top: 0.25rem;
       padding-bottom: 0.25rem;
-      &:hover {
-        color: var(--pst-color-primary);
-        border-style: none;
-        text-decoration-line: none;
-      }
+      @include link-style-text;
 
       &:focus {
         color: var(--pst-color-primary);
       }
     }
 
-    // Active page is always highlighted
+    // Active page is always underlined - using the same underline style as hover
+    // with the primary colour
```

Comment:
I'm not seeing this behavior on the PR build ("user guide" has no underline):

![Screenshot 2023-06-26 at 12-00-34 Accessibility — PyData Theme 0 13 4dev0 documentation](https://github.com/pydata/pydata-sphinx-theme/assets/1810515/c42b2983-750d-4c6c-b3be-10265ef5b318)

Reply:
I think we're talking about different things here. My understanding of the code comment was that the active section name would be teal with an underline *even when not hovering on it* --- my screenshot shows that it is teal but lacks an underline.  

Your screenshot seems to show what happens *when hovering* on the active section name --- secondary color with thick underline (which is expected and correct, but is not the problem I was trying to point out).

Reply:
aaaaah yes got confused - I noticed the underline has gone missing, and it is in my mind to fix it.
This is another example of having to fish changes in two places.